login_KA.feature

```diff
@@ -0,0 +1,144 @@
+Feature: log in Scoring 
```

Comment:
Page 33, p3: "Creating Feature file in format: Name of file.feature. Name of feature file and
keyword Feature: inside file should be the same"

Reply:
thank you! I corrected that